Source/ASTextNode2.mm

```diff
@@ -323,10 +320,17 @@ - (NSArray *)exclusionPaths
   return _textContainer.exclusionPaths;
```

Comment:
In the method above this (won't let me comment on it), setExclusionPaths:...is it possible for _textContainer to not be allocated yet? Should we call self.textContainer or something?

Reply:
No, the textContainer is created in -init and kept forever. 

A good way to think of it is the text container actually IS the text node in raw form. It stores many of the nodes properties and it's nicely copyable with atomic accessors. The text node itself is a bridge between the container and the rest of ASDK.